app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.wallpapers
+
+import android.view.View
+import mozilla.components.support.ktx.android.content.getColorFromAttr
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Provides access to available wallpapers and manages their states.
+ */
+class WallpaperManager(private val settings: Settings) {
```

Comment:
I really like this wrapper class for the functionality. Just thinking out loud, does this need any test coverage?

Reply:
For the moment this is more like a prototype, as we would like to collect feedback from nightly users while we shape the feature, as requirements at the moment are not fully completed, when we have some more concrete we will cover with tests :)